src/SourceLocator/DefaultSourceLocator.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace BetterReflection\SourceLocator;
+
+/**
+ * A simplification to avoid having to define the same three source locators
+ * all over the place. This should be able to locate most things in common
+ * environments.
+ */
+class DefaultSourceLocator extends AggregateSourceLocator
```

Comment:
Disagree on giving a default locator a new type, but if we go down this road:

 - implement `LocatorInterface`
 - don't inherit
 - build an `AggregateSourceLocator` internally